controllers/metering/controllers/metering_controller.go

```diff
@@ -42,10 +43,19 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/log"
 )
 
+func init() {
+	SealosSystemNamespace = os.Getenv("SEALOS_SYSTEM_NAMESPACE")
```

Comment:
The logic here may be a little unnecessary. This is unlikely to happen from the perspective of deployment.

Reply:
agree